tests/Altinn.DialogportenAdapter.Unit.Tests/Features/Command/Sync/StorageDialogportenDataMergerTest.cs

```diff
@@ -0,0 +1,44 @@
+using Altinn.DialogportenAdapter.WebApi.Features.Command.Sync;
+using Altinn.DialogportenAdapter.WebApi.Infrastructure.Storage;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.DialogportenAdapter.Unit.Tests.Features.Command.Sync;
+
+public class StorageDialogportenDataMergerTest
```

Comment:
Veldig bra at du la til en test her, er som du ser veldig tynt med det i adapteret.

Her trenger vi også en test for å håndtere at title, summary og knappelabels er innenfor begrensningene i Dialogporten. Hvis det som ligger i applicationtexts overskrides, må det kuttes og legges på "..."